### DISCUSSION TITLE
I have successfully made a roku app for free tv but i need people to help me maintain the links and switch them when they die

### DISCUSSION THREAD

**Discussion by CODERX24** [iptv-org/community#468] [Show and tell]
I have been switching links but frequently they die and I need a group of people to help me maintain the channels life
all my channels are hosted on github
link:
https://github.com/CODERX24/tv

**Comment by HNJAMeindersma** [comment]
Why do you need a group of people to do that for you when you can connect your app to data from iptv-org with some code? Since you've already been able to find this repo and write an app.

**Comment by CODERX24** [comment]
how do i do that?

**Comment by droidmarx** [comment]
How can I help you ?
